.gitignore

```diff
@@ -1,24 +1,64 @@
-# Compiled class file
-*.class
-
-# Log file
-*.log
-
-# BlueJ files
-*.ctxt
-
-# Mobile Tools for Java (J2ME)
-.mtj.tmp/
-
-# Package Files #
-*.jar
-*.war
-*.nar
-*.ear
-*.zip
-*.tar.gz
-*.rar
-
-# virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
-hs_err_pid*
-replay_pid*
+# Compiled class file
+*.class
+
+# Log file
+*.log
+
+# BlueJ files

```

Comment:
Are all of this really required? Can we please add only what we know will be part of the initial project and needs to be skipped? If there's something else we can add later.

Reply:
Fixed in, [e6e1e5e](https://github.com/ballerina-platform/static-code-analysis-tool/pull/2/commits/e6e1e5e0858412e46d9c195c808a59337071d050)